spec/models/user_spec.rb

```diff
@@ -0,0 +1,36 @@
+require './spec/rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(Name: 'Tom', Image: 'https://unsplash.com/photos/F_-0BxGuVvo', Bio: 'Teacher from Mexico.', PostCounter: 0) }
+
+  before { subject.save }
+
+  it 'should test for presence of name' do
+    subject.Name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'should test for empty name' do
+    subject.Name = ''
+    expect(subject).to_not be_valid
+  end
+
+  it 'should test for integer' do
+    subject.PostCounter = 'a'
+    expect(subject).to_not be_valid
+  end
+
+  it 'should test for integers lesser than 0' do
+    subject.PostCounter = -1
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] Unfortunately, this tests still fail on my end despite you making the changes requested by the previous reviewers. Kindly look into why this is the case

![image](https://user-images.githubusercontent.com/86318284/215291542-0737577d-6fd4-4427-b8ad-14e1dde6ec5f.png)